### PR TITLE
Added Via is-a-dev-cli

### PR DESCRIPTION
**Pull request by ajaein**
Added via the CLI

### CODE REVIEW

File: domains/.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "StudioYM",
+        "email": "null"
+      },
+      "record": {
+        "TXT": "d"
```
Comment (WilliamDavidHarrison) [inline domains/.json:8]: Invalid TXT

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please add a name for your domain.

File: domains/.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (WilliamDavidHarrison) [inline domains/.json:1]: Please delete this file

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please add a filename for the file named `.json`

**Review by wdhdev — CHANGES_REQUESTED** [review]
You can't use a Google IP

**Pull request closed without merging**